.github/workflows/codeql-analysis.yml

```diff
@@ -1,39 +1,33 @@
 name: "CodeQL Analysis"
+
 on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
   workflow_dispatch:
   schedule:
-    #        ┌───────────── minute (0 - 59)
-    #        │  ┌───────────── hour (0 - 23)
-    #        │  │ ┌───────────── day of the month (1 - 31)
-    #        │  │ │ ┌───────────── month (1 - 12 or JAN-DEC)
-    #        │  │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
-    #        │  │ │ │ │
-    #        │  │ │ │ │
-    #        │  │ │ │ │
-    #        *  * * * *
-    - cron: '30 1 * * *'
+    - cron: '32 5 * * 6'
```

Comment:
Should this be restored to `30 1 * * *`? I'm assuming that a daily scan is desired so at least the `6` should be changed to a `*`?

Reply:
Feel free to change it. I wanted to save a little amount of resources 😉